server/api/categories.go

```diff
@@ -362,3 +364,136 @@ func (a *API) handleUpdateCategoryBoard(w http.ResponseWriter, r *http.Request)
 	jsonBytesResponse(w, http.StatusOK, []byte("ok"))
 	auditRec.Success()
 }
+
+func (a *API) handleReorderCategories(w http.ResponseWriter, r *http.Request) {
+	// swagger:operation PUT /teams/{teamID}/categories/reorder handleReorderCategories
+	//
+	// Updated sidebar category order
+	//
+	// ---
+	// produces:
+	// - application/json
+	// parameters:
+	// - name: teamID
+	//   in: path
+	//   description: Team ID
+	//   required: true
+	//   type: string
+	// security:
+	// - BearerAuth: []
+	// responses:
+	//   '200':
+	//     description: success
+	//   default:
+	//     description: internal error
+	//     schema:
+	//       "$ref": "#/definitions/ErrorResponse"
+
+	vars := mux.Vars(r)
+	teamID := vars["teamID"]
+
+	ctx := r.Context()
+	session := ctx.Value(sessionContextKey).(*model.Session)
+	userID := session.UserID
+
+	requestBody, err := io.ReadAll(r.Body)
+	if err != nil {
+		a.errorResponse(w, r, err)
+		return
+	}
+
+	var newCategoryOrder []string
+
+	err = json.Unmarshal(requestBody, &newCategoryOrder)
+	if err != nil {
+		a.errorResponse(w, r, err)
+		return
+	}
+
+	auditRec := a.makeAuditRecord(r, "reorderCategories", audit.Fail)
```

Comment:
Should we add more information about what is going on in to the audit record here? (the same for the other API endpoint).

Reply:
Done.